dts/arm64/renesas/rcar_gen3_ca57.dtsi

```diff
@@ -28,6 +28,26 @@
 			     <GIC_PPI 10 IRQ_TYPE_LEVEL IRQ_DEFAULT_PRIORITY>;
 	};
 
+	reg_3p3v: regulator_3p3v {
+		compatible = "regulator-fixed";
+		regulator-name = "reg_3p3v";
+		regulator-min-microvolt = <3300000>;
+		regulator-max-microvolt = <3300000>;
+		regulator-always-on;
```

Comment:
I'm slightly confused about these regulator nodes:
- Should `always-on` be present, or `boot-on`? These regulators are not always enabled, and they are managed via the SDHC driver.
- Based on the changes to the fixed regulator driver and the lack of an enable pin, are these regulator nodes just being used to determine the voltage the SD host controller supports? If so, we should just make this a property of the host controller binding.

Reply:
Ok- I'm not familiar with the Rcar boards, but given that #58411 is merged, can this PR be updated to use that driver for either of these boards?

Reply:
Ah, I got this. This is for eMMC, when regulators are indeed fixed. It it strange that you didn't mentioned in the commit message, that you are adding DTS definitions for eMMC.

Reply:
Nothing seems strange to me. Feel free to walk through Zephyr commits in the 'dts' directory and look at the commit descriptions to see how other people have done it.